marketing_tool.py

```
@@ -0,0 +1,1388 @@
+    header_panel = Panel(
+        Align.center(header_text),
+        border_style=COLOR_ACCENT_AMBER,
+        box=box.ROUNDED,
+        style=BACKGROUND_STYLE,
+        padding=(0, 2),
+    )
+    console.print(header_panel)
+    console.print()
+    console.print(layout)
+    console.print()
+    console.print(f"[dim]The system has handled the dirty work (segments, scheduling, syncs) automatically.[/dim]")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)